automation/workflow/hook.go

```diff
@@ -0,0 +1,24 @@
+package workflow
+
+import "github.com/qri-io/qri/event"
+
+// "encoding/json"
+
+// HookType is the type of hook
+type HookType string
+
+// A Hook determines under what circumstances its `hook.Event()` should be
+// emitted, and what the event payload should be.
+type Hook interface {
+	// json.Marshaler
+	// json.Unmarshaler
+	ID() ID
+	IsEnabled() bool
+	Enable() error
+	Disable() error
```

Comment:
Combine these into a setter: `SetEnabled(enabled bool) error`

Reply:
This is great ty